xtp/src/pyxtp/CMakeLists.txt

```
@@ -30,24 +20,25 @@ target_link_libraries(xtp_binds PUBLIC VOTCA::votca_xtp)
 if(NOT CMAKE_INSTALL_PYTHON_LIBDIR)
   set(CMAKE_INSTALL_PYTHON_LIBDIR "${CMAKE_INSTALL_LIBDIR}/python${Python_VERSION_MAJOR}.${Python_VERSION_MINOR}/site-packages")
 endif()
-install(TARGETS xtp_binds LIBRARY DESTINATION ${CMAKE_INSTALL_PYTHON_LIBDIR}/pyxtp)
-add_library(VOTCA::xtp_binds ALIAS xtp_binds)
 
-file(GLOB_RECURSE PYXTP_FILES pyxtp/*.py)
-message(STATUS "PYTHON INSTALL: ${CMAKE_INSTALL_PYTHON_LIBDIR}")
-install(FILES ${PYXTP_FILES} DESTINATION ${CMAKE_INSTALL_PYTHON_LIBDIR}/pyxtp)
+install(TARGETS xtp_binds LIBRARY DESTINATION ${CMAKE_INSTALL_PYTHON_LIBDIR}/pyxtp_binds)
+add_library(VOTCA::xtp_binds ALIAS xtp_binds)
 
-# copy py files to build dir, so CMAKE_CURRENT_BINARY_DIR is a complete py module with script and xtp_binds
-foreach(_PYFILE ${PYXTP_FILES})
-  get_filename_component(_FILE "${_PYFILE}" NAME)
-  add_custom_command(OUTPUT ${CMAKE_CURRENT_BINARY_DIR}/${_FILE}
-    COMMAND ${CMAKE_COMMAND} -E copy_if_different ${_PYFILE} ${CMAKE_CURRENT_BINARY_DIR}/${_FILE}
-    DEPENDS ${_DEP}
+install (CODE "
+    message(STATUS \"Installing Python package with pip from ${CMAKE_CURRENT_SOURCE_DIR}...\")
+    execute_process(
+    COMMAND ${Python_EXECUTABLE} -m pip install --upgrade --target ${CMAKE_INSTALL_PYTHON_LIBDIR} ${CMAKE_CURRENT_SOURCE_DIR}
```

```suggestion
    COMMAND ${Python_EXECUTABLE} -m pip install --upgrade --target $ENV{DESTDIR}${CMAKE_INSTALL_PYTHON_LIBDIR} ${CMAKE_CURRENT_SOURCE_DIR}
```
or something similar.

---

What does the DESTDIR envvar point to? Is that related to the sphinx build that keeps blocking me?

---

The default is empty, but it is used for packaging:
https://cmake.org/cmake/help/latest/envvar/DESTDIR.html